app/components/mandatarissen/update-state.hbs

```diff
@@ -57,6 +57,46 @@
       >De gekozen datum moet na de startdatum liggen van het huidige mandaat.</AuAlert>
     {{/unless}}
   </div>
+  {{#unless this.isTerminatingMandate}}
+    <div>
+      <AuLabel>
+        Beleidsdomeinen
+      </AuLabel>
+      <Mandatarissen::BeleidsdomeinSelectorWithCreate
+        @beleidsdomeinen={{this.selectedBeleidsdomeinen}}
+        @onSelect={{this.updateBeleidsdomeinen}}
+      />
+      <AuHelpText>Enkel voor burgemeester en schepenen</AuHelpText>
```

Comment:
If this is true, this is really something we should clarify in other places as well, or limit with form restrictions or something like that.

Reply:
i agree that we should limit this, i'm doing a trial for things like this in my final story in this 'saga', but didn't want to add it to existing things yet